testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite1.java

```diff
@@ -31,7 +31,7 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
     {
         return ImmutableList.of(
                 testOnEnvironment(EnvMultinode.class)
-                        .withExcludedGroups("large_query", "storage_formats", "storage_formats_detailed", "profile_specific_tests", "tpcds", "hive_compression", "azure")
+                        .withExcludedGroups("large_query", "storage_formats", "storage_formats_detailed", "profile_specific_tests", "tpcds", "hive_compression", "azure", "hms_only")
```

Comment:
I think this is not needed. I may be wrong but I think hms_only was not run with Suite1.
And if I am wrong I'd extract this to a separate commit

Reply:
@homar I will extract to a separate commit. The PR was failing previously without this change because the tests I added were running in `Suite1`.